utils/utils.js

```diff
@@ -1,3 +1,5 @@
 module.exports = {
     Random: require('./random.js'),
+    Average: require('./average.js'),
+    BayesUtils: require('./utils_bayes.js')
```

Comment:
Could be just Bayes. The file could be called index.js (since it sums up all others).